src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -413,7 +413,7 @@ module TcRecdUnionAndEnumDeclarations =
         let attrsForProperty, attrsForField = attrs |> List.partition (fun (attrTargets, _) -> (attrTargets &&& AttributeTargets.Property) <> enum 0) 
         let attrsForProperty = (List.map snd attrsForProperty) 
         let attrsForField = (List.map snd attrsForField)
-        let tyR, _ = TcTypeAndRecover cenv NoNewTypars CheckCxs ItemOccurence.UseInType env tpenv ty
+        let tyR, _ = TcTypeAndRecover cenv NoNewTypars CheckCxs ItemOccurence.UseInType WarnOnIWSAM.Yes env tpenv ty
```

Comment:
It's probably too late for this, but just saying that IWSAM would be completely mysterious for a random reader. Like, you can add this to the coding standards doc with the abbreviations, but I'd just prefer a long(er) name, because... why not?